### PR TITLE
Update documentation with examples

### LINKED ISSUE #6224: Complete documentation for Ballerina Guidewire InsuranceNow connector
**Description:**
> $subject

### PR DESCRIPTION
**Pull request by ayeshLK**
Fixes: [#6224](https://github.com/ballerina-platform/ballerina-library/issues/6224)

## Examples
N/A

## Checklist
- [X] Linked to an issue
- [ ] ~Updated the changelog~
- [ ] ~Added tests~
- [ ] ~Updated the spec~
- [ ] ~Checked native-image compatibility~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,108 @@
...
+```
+
+### Step 3: Invoke the connector operations
+
+Now, utilize the available connector operations.
+
+#### List applications
+
+```ballerina
+insnow:ListApplication applications = check insuranceNow->/applications();
```
Comment (DimuthuMadushan) [inline README.md:56]:
```suggestion
insnow:ListApplication applications = check insNow->/applications();
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,108 @@
...
+        },
+        {
+            id: "tag-auto-insurance",
+            name: "Auto Insurance",
+            tagTemplateIdRef: "template-auto-ins"
+        }
+    ],
+    templateId: "template-insurance-application"
+};
+_ = check insuranceNow->/applications/["123"]/documents.post(attachment);
```
Comment (DimuthuMadushan) [inline README.md:96]:
```suggestion
_ = check insNow->/applications/["123"]/documents.post(attachment);
```

File: README.md [hunk 3]
```diff
@@ -6,6 +6,108 @@
...
+
+## Setup guide
+
+To use the Ballerina Guidewire InsuranceNow connector, access to the Guidewire API is required. Access is granted to partners or customers of the Guidewire platform. To become a partner or customer, reach out to Guidewire's sales team via the [Guidewire website](https://www.guidewire.com/contact-us/). For detailed instructions on obtaining authentication/authorization keys, please go through the [portal development guide](https://docs.guidewire.com/cloud/in/20233/portaldev/PortalDevelopment/topics/c_overview_portal_development.html).
+
+## Quickstart
+
+To use the Guidewire InsuranceNow connector in your Ballerina application, modify the `.bal` file as follows:
+
+### Step 1: Import the connector
```
Comment (Nuvindu) [inline README.md]:
```suggestion
### Step 1: Import the module
```

File: README.md (hunk 1 above)
Comment (ayeshLK) [reply README.md:56]: `insuranceNow` provide a clear meaning rather than compacted version. Hence, used this convention.

Comment (DimuthuMadushan) [reply README.md:56]: You have initialized the client as `insNow` in previous section.

**Review by Nuvindu — DISMISSED** [review]

File: README.md (hunk 1 above)
Comment (ayeshLK) [reply README.md:56]: Fixed with [39ffe40e0f70f5f3af651fee10e11e3952c9d49c](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/14/commits/39ffe40e0f70f5f3af651fee10e11e3952c9d49c)

File: README.md (hunk 2 above)
Comment (ayeshLK) [reply README.md:96]: Fixed with [39ffe40e0f70f5f3af651fee10e11e3952c9d49c](https://github.com/ballerina-platform/module-ballerinax-guidewire.insnow/pull/14/commits/39ffe40e0f70f5f3af651fee10e11e3952c9d49c)

**Review by DimuthuMadushan — APPROVED** [review]

**Pull request merged**